apps/controllerx/devices/aqara.py

```diff
@@ -26,3 +26,29 @@ def get_z2m_actions_mapping(self):
             "right_double": Light.CLICK_BRIGHTNESS_UP,
             "right_long": Light.CLICK_BRIGHTNESS_DOWN,
         }
+
+
+class MFKZQ01LMController(LightController):
```

Comment:
Since there might be a implementation for this controller as a media player controller, I would change the class to `MFKZQ01LMLightController`.
```suggestion
class MFKZQ01LMLightController(LightController):
```